FuelSDK-CSharp/ETClient.cs

```diff
@@ -148,6 +151,53 @@ public ETClient(NameValueCollection parameters = null, RefreshState refreshState
                 }
         }
 
+        private void ValidateRequiredConfigValues()
```

Comment:
Please add a check for the RestEndpoint also.

Reply:
Method refactored.